auth/credentials/oauth.go

```diff
@@ -151,7 +138,7 @@ func parseToken(accessToken string) (*Token, error) {
 
 	expiry := time.Unix(tokenData.Exp, 0)
 	if time.Now().After(expiry) {
-		return nil, errors.New("Token has expired")
+		return nil, errors.New("atlas cloud Access Token has expired")
```

Comment:
```suggestion
		return nil, errors.New("Atlas Cloud Access Token has expired")
```

Reply:
FYI: this would normally fail linting but we do not have error/comments linting enabled in the repository.